src/ameba/rule/style/redundant_begin.cr

```diff
@@ -90,7 +90,7 @@ module Ameba::Rule::Style
     end
 
     private def redundant_begin_in_expressions?(node)
-      node.keyword == :begin
+      !!(node.keyword.try &.begin?)
```

Comment:
```suggestion
      !!node.keyword.try(&.begin?)
```

Reply:
Matter of style. I think both versions are acceptable